shared/pinentry/index.render.js.flow

```diff
@@ -0,0 +1,16 @@
+// @flow
+import React, {Component} from 'react'
+import type {Feature} from '../constants/types/flow-types'
+
+export type Props = {
+  onSubmit: (passphrase: string, features: {[key: string]: boolean}) => void,
+  onCancel: () => void,
+  features: {[key: string]: Feature},
+  prompt: string,
+  retryLabel: string,
+  cancelLabel: ?string,
+  submitLabel: ?string,
+  windowTitle: string
```

Comment:
Looks like cancelLabel, submitLabel, and windowTitle aren't used?

Maybe we should delete those.